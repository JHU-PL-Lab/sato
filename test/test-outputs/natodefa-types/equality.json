[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "a" ],
          "err_match_val": "true",
          "err_match_expected": "Integer",
          "err_match_actual": "Boolean"
        }
      ],
      [
        "Error_match",
        {
          "err_match_aliases": [ "b" ],
          "err_match_val": "2000",
          "err_match_expected": "Boolean",
          "err_match_actual": "Integer"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "a == b",
    "err_steps": 182
  }
]