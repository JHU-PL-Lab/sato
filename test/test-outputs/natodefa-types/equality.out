Type errors for:
- Input sequence : []
- Found at expr  : a == b
--------------------
* Value    : a = true
* Expected : Integer
* Actual   : Boolean
--------------------
* Value    : b = 2000
* Expected : Boolean
* Actual   : Integer
2 errors found. No further control flows exist.
